### ISSUE TITLE
Array src is not working

### ISSUE THREAD

**Issue by aspirisen** [damorej-at-theinstitutes/webpack-concat-files-plugin#20]
There is an issue when you pass `src` as array. Looks like because `replacePathSeparator` from `src/util/path-separators.js` expects plain string.

```
node_modules/webpack-concat-files-plugin/src/util/path-separators.js:12
  return filepath.split(path.sep).join(newSeparator);
                  ^
TypeError: filepath.split is not a function
```

**Comment by damorej-at-theinstitutes** [comment]
Hi @aspirisen, thanks for creating this issue!

I just pushed some changes to the `develop` branch to add tests for this and to resolve the issue. It seems to be working on my end, but would you be able to confirm that it fixes your issue?

If everything seems to be working, I'll prepare a release for this as `v0.5.1`.

Thanks again!

**Comment by aspirisen** [comment]
Yeah that should help, thanks 😊

**Comment by damorej-at-theinstitutes** [comment]
Great! I just released `v0.5.1` and published on NPM. I'm going to close this issue now, but don't hesitate to post here again or open a new issue if you encounter any other problems. Thanks again for opening this issue @aspirisen!
